client/src/components/Restaurant.jsx

```diff
@@ -0,0 +1,37 @@
+import Map from './Map';
+
+const Restaurant = ({ restau }) => {
+  return (
+    <div className='restauContainer'>
+      {/* first, make sure we have data - redirect home otherwise 
+          unfortunately, this is not working. */}
+      {/* {restau === [] ? <Redirect to='/' /> : undefined} */}
```

Comment:
verified this too. it does print the loading message from below however. Better than crashing the application!